iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractTwoWindingsTransformerTest.java

```diff
@@ -212,6 +231,92 @@ public void transformerNotInSameSubstation() {
         assertTrue(e.getMessage().contains("the 2 windings of the transformer shall belong to the substation"));
     }
 
+    @Test
+    public void testTwoWindingsTransformersCopier() {
```

Comment:
Could you also modified `AbstractSubnetworksCreationTest.testTwoWindingsTransformersCreation()` to test that when a subnetwork is specified at creation, the resulting two-winding transformer is affected to the subnetwork?

Reply:
Thanks!